lms/djangoapps/certificates/models.py

```diff
@@ -436,6 +436,23 @@ def get_certificate_invalidations(cls, course_key, student=None):
             })
         return data
 
+    @classmethod
+    def is_certificate_invalid(cls, course_key, student):
+        """Check that certificate for a student is invalidated or not.
```

Comment:
`Check that whether the student in the course has been invalidated for receiving certificates`